cmd/eval-dev-quality/cmd/evaluate_test.go

```diff
@@ -54,7 +57,8 @@ func TestEvaluateExecute(t *testing.T) {
 			"--repository", "golang/plain",
 		},
 
-		ExpectedOutputContains: `Evaluation score for "symflower/symbolic-execution" ("code-no-excess"): score=6, coverage-statement=1, files-executed=1, response-no-error=1, response-no-excess=1, response-not-empty=1, response-with-code=1`,
+		ExpectedOutputContains:      `Evaluation score for "symflower/symbolic-execution" ("code-no-excess"): score=6, coverage-statement=1, files-executed=1, response-no-error=1, response-no-excess=1, response-not-empty=1, response-with-code=1`,
```

Comment:
Please validate the .csv as well with the same logic. there should be only models, languages, repositories in there that we filtered for.